extensions/cornerstone/src/panels/PanelMeasurementWithNodeComponent.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useRef } from 'react';
+import debounce from 'lodash.debounce';
+import { useMeasurements } from '../hooks/useMeasurements';
+
+import { utils } from '@ohif/core';
+import { PanelAccordion } from '../components/CollapsibleStudySummaryFromMetadata';
+import { StudySummaryFromMetadata } from '../components/StudySummaryFromMetadata';
+import NodeComponent from './NodeComponent';
+import { MeasurementTable } from './TableForNodeComponent';
+
+const { groupByStudy } = utils.MeasurementGroupings;
+const { filterAdditionalFindings, filterAny } = utils.MeasurementFilters;
+
+export type withAppAndFilters = withAppTypes & {
+  measurementFilter?: (item) => boolean;
+};
+
+export default function PanelMeasurement({
+  servicesManager,
+  commandsManager,
+  measurementFilter = filterAny,
+}: withAppAndFilters): React.ReactNode {
+  const { measurementService } = servicesManager.services;
+
+  const measurementsPanelRef = useRef(null);
+  const additionalFilter = filterAdditionalFindings(measurementService);
```

Comment:
The additional filter should be part of the grouping function for the nested child element.